x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -54,14 +54,14 @@ export interface SetupPlugins {
   licensing: LicensingPluginSetup;
   security?: SecuritySetup;
   spaces?: SpacesSetup;
-  taskManager: TaskManagerSetupContract;
+  taskManager?: TaskManagerSetupContract;
   ml?: MlSetup;
   lists?: ListPluginSetup;
 }
 
 export interface StartPlugins {
-  ingestManager: IngestManagerStartContract;
-  taskManager: TaskManagerStartContract;
+  ingestManager?: IngestManagerStartContract;
```

Comment:
Since both `ingestManager` and `taskManager` are required plugins, these shouldn't be typed as optional here and the downstream null checks are unnecessary. Is this a temporary situation, or can we clean this up?